pkg/db/db.go

```diff
@@ -0,0 +1,117 @@
+package db
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/jackc/pgconn"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/config"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/types"
+)
+
+func getConnectionString(db *config.DB) string {
```

Comment:
Why are we coupling the DB module with the program's config structure?

Reply:
i moved this to config to return a string with connection details to connect to db